docs/source/tutorials/multilabel_classification.ipynb

```diff
@@ -0,0 +1,303 @@
+    "\n",
+    "<div  class=markdown markdown=\"1\" style=\"background:white;margin:16px\">  \n",
+    "    \n",
+    "```ipython3 \n",
+    "from cleanlab.filter import find_label_issues\n",
+    "\n",
+    "noise_idx = find_label_issues(\n",
+    "        labels=dataset[\"labels\"],\n",
+    "        pred_probs=dataset[\"pred_probs\"],\n",
+    "        multi_label=True,\n",
+    "        return_indices_ranked_by='normalized_margin',\n",
+    "    )\n",
+    "```\n",
+    "\n",
+    "    \n",
+    "</div>\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6a6261a3-6ea1-44a6-ac91-d375c8aa5535",
+   "metadata": {},
+   "source": [
+    "## 1. Install required dependencies and download data\n",
+    "\n",
+    "```ipython3\n",
+    "!pip install cleanlab\n",
+    "!pip install torchvision\n",
+    "\n",
+    "\n",
+    "# Make sure to install the version corresponding to this tutorial\n",
+    "# E.g. if viewing master branch documentation:\n",
+    "#     !pip install git+https://github.com/cleanlab/cleanlab.git\n",
+    "\n",
+    "```\n",
+    "Kaggle Dataset source: [Kaggle CelebA](https://www.kaggle.com/datasets/jessicali9530/celeba-dataset)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e8ff5c2f-bd52-44aa-b307-b2b634147c68",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!wget -nc 'https://cleanlab-public.s3.amazonaws.com/MultilabelClassification/pred_probs.npy'\n",
+    "!wget -nc 'https://cleanlab-public.s3.amazonaws.com/MultilabelClassification/data.csv'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "354f5f64-8a91-426f-866b-d27a93477748",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!wget -nc -O 'archive.zip' 'https://storage.googleapis.com/kaggle-data-sets/29561/37705/bundle/archive.zip?X-Goog-Algorithm=GOOG4-RSA-SHA256&X-Goog-Credential=gcp-kaggle-com%40kaggle-161607.iam.gserviceaccount.com%2F20221102%2Fauto%2Fstorage%2Fgoog4_request&X-Goog-Date=20221102T043448Z&X-Goog-Expires=259200&X-Goog-SignedHeaders=host&X-Goog-Signature=4a8303ca846eaae80b62ff3705ad6b86de91724f8d76b4373083c4c33225314b6c43af9bab79e32100af51aa02e230a1eaa81ae33aea94680fbb118fb74dd6bf4d2a9426809a43ce699da806f156781c98f5076845d8936c304657265ecaa323ba0a369aa7d436916a6a87ba8a5873865da1b30052c9f128f2005fa12e1526850a79fe96e74aa698bc4ce59b3822d159b22ece231b7246d7800b685c5e9974e7837dfb4d2d7c18147749443bdbe9ae44f5db3e045f25774df6d464c7f0762279fea7b201ce76ceb15d1301f4354112db105c108dd5564f3a165f9f2af1fb581f5fa6d5b471a194007290b03637a38105c09a3c6f2aebb26e3df2fccf49d349a6'\n",
+    "!unzip archive.zip"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "74ca37ea-cc7c-42f2-9330-dac806b015c3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "from cleanlab.filter import find_label_issues\n",
```

Comment:
let's import cleanlab functions after other imports in all tutorials

Reply:
Fixed: https://github.com/cleanlab/cleanlab/pull/517/commits/7aa53405e8abc5f3f0aa2f2a3a96f90f21516db1